src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -108,13 +111,28 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
       >
         Implement Change Request
       </NERButton>
-      <Menu open={dropdownOpen} anchorEl={anchorEl} onClose={handleDropdownClose}>
+      <Menu
+        open={dropdownOpen}
+        anchorEl={anchorEl}
+        onClose={handleDropdownClose}
+        anchorOrigin={{
```

Comment:
Right now, your dropdown menu is being displayed so that it aligns with the 'Implement Change Request' button on the right side. Before, the dropdown menu was displayed so that it aligned with this button on the left side, so you might want to change it back. You would have to change the 'horizontal' field for both anchorOrigin and transformOrigin I'm pretty sure.

Reply:
Yeah it wasn't originally part of the ticket, but I kinda like it this way, it aligns with the arrow on the right, so we can just leave it the way you have it here